tox.ini

```diff
@@ -12,12 +13,13 @@ envlist =
 deps =
     .[test]
     zope.testrunner
+    coverage
 commands =
-    zope-testrunner --test-path=src {posargs:-vc}
+    coverage run -m zope.testrunner --test-path=src {posargs:-vc}
```

Comment:
Having `coverage run` in the general `testenv` breaks parallel test runs with `detox`, in the sense that the coverage results are meaningless. To fix that we'd need to specify different coverage file names per env, and then have an env that does `coverage combine`. But there's also no 'coverage report --fail-under' so I'm not sure why we're even doing this here. (I know, it's to make things simpler in .travis.yml.)

Reply:
Hmm, it turns out that doing coverage from tox does *not* play well with coveralls, at least as presently set up. Attempting to view a file and see coverage data [responds with "source not available"](https://coveralls.io/builds/19400314/source?filename=.tox%2Fpy37%2Flib%2Fpython3.7%2Fsite-packages%2Fzope%2Ftestbrowser%2Fcookies.py) because it's trying to drill into `.tox/`. This could possibly be fixed using 'usedevelop=True' in tox.ini.

Reply:
Alternatively it can be fixed by specifying equivalent source paths in `.coveragerc`, e.g. this is from zope.annotation's:
```
[paths]
source =
    src/
    .tox/*/lib/python*/site-packages/
    .tox/pypy*/site-packages/
```

Reply:
I don't know if that would work. The coverage is getting recorded (although possibly not merged, which that *might* fix---but zope.annotation doesn't use tox on travis so we can't be sure). The problem is that coveralls can't find the source on github to use to display the coverage data: 

> The file ".tox/py37/lib/python3.7/site-packages/zope/testbrowser/cookies.py" isn't available on github. Either it's been removed, or the repo root directory needs to be updated.

Reply:
At this point I'm thinking it might be easier to go back to dropping `tox` and hooking up flake8/sphinx some other way.

E.g. we could run flake8 and sphinx against ALL python versions, instead of just one?  Sequentially, after zope-testrunner.